usagov/vote-gov-drupal
# Feature/vote 2085/high contrast register to vote

## Jira ticket

[VOTE-2085](https://cm-jira.usa.gov/browse/VOTE-2085)

## Description

Implement high contrast styles to the Register to vote component.

## Deployment and testing

### Post-deploy steps

1. lando retune
2. cd into theme - npm run build

### QA/Testing instructions

1. Sign in as admin
2. Navigate to http://vote-gov.lndo.site/register
3. Toggle the high contrast button, verify all the high-contrast styling matches the design
4. Toggle back to default, verify none of the high contrast styles are being applied

## Checklist for the Developer




## Checklist for the Peer Reviewers
